test/chpldoc/enum.doc.good

```diff
@@ -22,8 +22,42 @@ or
    This is a color enum. 
 
 
```

Comment:
Can we get it so that we only generate one line in between the enum's documentation and these comment lines?  Based on the other .good insertions, this might mean modifying the documentation in the test itself

Reply:
Note that you'd want to do that for the other tests as well